subjects/blockchain/node-dashboard/audit/README.md

```diff
@@ -0,0 +1,37 @@
+## Node Dashboard audit
+
+##### Read the documentation
+###### Does the README file contains the instructions to launch the project?
+###### Are two different clients launched?
+###### Is clique the consensus algorithm?
+
+##### Launch the network with the `launch-network` and display its status with `infos`
+###### Does the command launch the five nodes?
+###### Does the interface display the latest block of each node?
+###### Does the interface display their Ethereum address and balance?
+###### Does the interface display the CPU and memory consumption of each node?
+
+##### Launch scenario 0
+###### Does Alice, Bob and Cassandra have a positive balance of ETH?
+
+##### Launch scenario 1
+###### Does the command provide feedback?
+###### Does the `infos` command show the updated ETH balance? 
+###### Does the `infos` command displays transactions in the mempool?
+
+##### Launch scenario 2
+###### Does the command provide feedback?
+###### Do Driss and Elena appear to have received 1000 BY tokens?
+
+##### Launch scenario 3
+###### Does the command provide feedback?
+###### Does the `infos` command show the updated balance of Elena receiving one ETH?
+
+##### Run `temporary-failure Alice` and launch `infos`
+###### Is the node disabled in the `infos` output?
+###### Does it come back online after 40 seconds?
+###### Does it appear to be up to date a couple of minutes later?
+
+#### Bonus 
+###### +Does the `-u` option allow you to launch each command with a regular update?
+###### +Does the tool offer the possibility of launching a node connected to one of the testnet networks?
```

Comment:
Usually there is a line spacing between all the lines in the audits - it seems to not affect the final rendering, but it would be nice to keep the style consistent in my opinion

Reply:
done